doc/nrf/ug_nrf7002.rst

```diff
@@ -0,0 +1,248 @@
++-----------+------------+----------------------+
+| UARTE     | on-chip    | serial               |
++-----------+------------+----------------------+
+| USB       | on-chip    | usb                  |
++-----------+------------+----------------------+
+| WDT       | on-chip    | watchdog             |
++-----------+------------+----------------------+
+
+The ``nrf7002dk_nrf5340_cpunet`` board configuration supports the following hardware features:
+
++-----------+------------+----------------------+
+| Interface | Controller | Driver/Component     |
++===========+============+======================+
+| CLOCK     | on-chip    | clock_control        |
++-----------+------------+----------------------+
+| FLASH     | on-chip    | flash                |
++-----------+------------+----------------------+
+| GPIO      | on-chip    | gpio                 |
++-----------+------------+----------------------+
+| I2C(M)    | on-chip    | i2c                  |
++-----------+------------+----------------------+
+| MPU       | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| NVIC      | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| RADIO     | on-chip    | Bluetooth,           |
+|           |            | ieee802154           |
++-----------+------------+----------------------+
+| RTC       | on-chip    | system clock         |
++-----------+------------+----------------------+
+| RTT       | Segger     | console              |
++-----------+------------+----------------------+
+| QSPI      | on-chip    | qspi                 |
++-----------+------------+----------------------+
+| SPI(M/S)  | on-chip    | spi                  |
++-----------+------------+----------------------+
+| UARTE     | on-chip    | serial               |
++-----------+------------+----------------------+
+| WDT       | on-chip    | watchdog             |
++-----------+------------+----------------------+
+
+Other hardware features are not supported by the |NCS| kernel.
+
+Connections and IOs
+===================
+
+The following are the connections and IOs supported by the development kit.
+
+LED
+---
+
+* LED1 (green) = P1.06
+* LED2 (green) = P1.07
+
+Push buttons
+------------
+
+* BUTTON1 = SW1 = P1.08
+* BUTTON2 = SW2 = P1.09
+* BOOT = SW5 = boot/reset
+
+Wi-Fi control
+-------------
+
+* BUCKEN = P0.12
+* IOVDD CONTROL = P0.31
+* HOST IRQ = P0.23
+* COEX_REQ = P0.28
+* COEX_STATUS0 = P0.30
+* COEX_STATUS1 = P0.29
+* COEX_GRANT = P0.24
+
+Security components
+===================
+
+The following security components are available:
+
+* Implementation Defined Attribution Unit (`IDAU`_) on the application core.
+
+  The IDAU is implemented with the System Protection Unit and is used to define secure and non-secure memory maps.
+  By default, all of the memory space (Flash, SRAM, and peripheral address space) is defined to be secure accessible only.
+
+* secure boot.
```

Comment:
```suggestion
* Secure Boot.
```
This suggestion was already present in a previous comment

Reply:
@peknis  suggested not to use upper case, so, shall I go with `Secure boot`?

Reply:
In this case, as a list item, use sentence case, so the first word starting with a capital letter - > Secure